SimG4Core/Application/src/StackingAction.cc

```diff
@@ -418,8 +415,7 @@ void StackingAction::initPointer() {
     if (savePDandCinTracker &&
         (rname == "BeamPipe" || rname == "BeamPipeVacuum" || rname == "TrackerPixelSensRegion" ||
          rname == "TrackerPixelDeadRegion" || rname == "TrackerDeadRegion" || rname == "TrackerSensRegion" ||
-         rname == "FastTimerRegionBTL" || rname == "FastTimerRegionETL" || rname == "FastTimerRegionSensBTL" ||
-         rname == "FastTimerRegionSensETL")) {
+         rname == "FastTimerRegion" || rname == "FastTimerRegionSensBTL" || rname == "FastTimerRegionSensETL")) {
```

Comment:
@civanch this code undo what I added in my earlier PR, the new BTL and ETL regions should be kept, or the transition logic will not be effctive. As you may see there are changes in MTD tests which should not be there

Reply:
It is already fixed.